test/unit-test/CMakeLists.txt

```diff
@@ -262,9 +261,9 @@ endforeach()
 # They are copied from https://github.com/google/u2f-ref-code/tree/master/u2f-tests/HID
 set(U2F_TESTS
    u2f_standard
-   "-Wl,--wrap=workflow_unlock"
+   ""
    u2f_hid
-   "-Wl,--wrap=workflow_status_create"
+   ""
```

Comment:
Since there are no args anymore, might as well remove the empty strings and simplify the loop below?

```
set(U2F_TESTS
   u2f_standard
   u2f_hid
)
foreach(TEST_NAME ${U2F_TESTS})
  set(EXE test_${TEST_NAME})
  target_link_libraries(${EXE} PRIVATE
    ...
    u2f-util
    ${CMOCKA_LIBRARIES}
)
```

All the `math()` and `list()` would be gone. Simpler.

You'll say, but we might need them later, and I'd reply with "we can get them back from git commit history". :)

Reply:
Done :P